pkg/appsec/config/config.go

```diff
@@ -0,0 +1,68 @@
+package config
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/pkg/errors"
+)
+
+// AgentConfig represents the agent configuration API.
+type AgentConfig interface {
+	GetBool(key string) bool
+	GetString(key string) string
+	GetInt(key string) int
+}
+
+const (
+	// defaultIntakeURLTemplate specifies the string template allowing to obtain
+	// the intake URL from a given site.
+	defaultIntakeURLTemplate = "https://appsecevts-http-intake.logs.%s/v1/input"
+	// defaultSite is the intake site applied by default to
+	// defaultIntakeURLTemplate
```

Comment:
I'd recommend documenting only _what it is_ not _where it is used_.

Reply:
I think it does with "intake site applied by default to [the default intake url template] defaultIntakeURLTemplate.
I tried to make it more verbose. Let me know if anything else.

Reply:
What I meant is:
```
// defaultSite is the default site used to target the intake.
```
Or similar. No need to mention the other constant, and how it is used. As far as we know, it may be used anywhere and it's not restricted to that purpose. But do as you please, I don't feel strongly here.

Reply:
I see. Fixed.